pkg/kubelet/dockertools/labels.go

```diff
@@ -73,6 +73,10 @@ func GetPodName(info *cadvisorapiv2.ContainerInfo) string {
 	return info.Spec.Labels[kubernetesPodNameLabel]
 }
 
+func GetPodUID(info *cadvisorapiv2.ContainerInfo) string {
```

Comment:
Yeah. I think that would be better. I'm happy to make the change in this PR if you want.